src/resolve/connectionResolver.ts

```diff
@@ -125,93 +120,60 @@ export class ConnectionResolver {
       apiVersion: this.connection.getApiVersion(),
     };
   }
+}
 
-  private async listMembers(query: ListMetadataQuery): Promise<RelevantFileProperties[]> {
-    let members: RelevantFileProperties[] = [];
-
-    const pollingOptions: PollingClient.Options = {
-      frequency: Duration.milliseconds(1000),
-      timeout: Duration.minutes(3),
-      poll: async (): Promise<StatusResult> => {
-        const res = ensureArray(await this.connection.metadata.list(query));
```

Comment:
Have our connections just become more resilient to remove this polling? I don't understand why we're doing this to begin with

Reply:
yeah, part of the jsforce stuff that Cristian did https://github.com/jsforce/jsforce/pull/1407.

I think the retry logic was using pollingClient to work around those network issues.

Reply:
@WillieRuemmele Shane is correct, I added the pollingClient logic here:
https://github.com/forcedotcom/source-deploy-retrieve/pull/941

jsforce retries on the same errors that pollingClient and more.